### ISSUE TITLE
FYI: OS/2 Support is back in SDL 2 mainline

### ISSUE THREAD

**Issue by LowLevelMahn** [dosbox-staging/dosbox-staging#671]
https://hg.libsdl.org/SDL/rev/ac0117d933f7

only AmigaOS is missing to get on par with SDL 1

**Comment by dreamer** [comment]
Good to know, but I don't care. Not a single person asked about OS/2 support in DOSBox Staging since we removed it.

Since that time we had people interested in Haiku, FreeBSD people working on packaging us on their own, AmigaOS people contributing to our development, recently we had interest in improving support for Purism (plus some kind words from people testing DOSBox Staging on niche ARM hardware), I am personally interested in making Redox OS happen perhaps. Not a single person asked for OS/2 though.

If anyone is interested - here's what we removed: 5f9ac5eeab62e58063c2a760f6d5fdd2513b476f, feel free to work on reverting it and we will consider merging the PR (if anyone will send it).
